src/quo2/foundations/colors.cljs

```diff
@@ -60,6 +60,7 @@
 
 ;;Blur
 (def neutral-5-opa-70 (alpha neutral-5 0.7))
+(def neutral-80-blur-opa-80 "rgba(25,36,56,0.8)")
```

Comment:
Could you explain why we can't use `alpha` with some (maybe yet to be defined) var color?

Reply:
More context here: https://github.com/status-im/status-mobile/pull/15169/files#r1115648577

Reply:
Thanks @briansztamfater